pkgs/applications/editors/emacs/generic.nix

```diff
@@ -67,18 +67,18 @@ in stdenv.mkDerivation {
 
     # Make native compilation work both inside and outside of nix build
     (lib.optionalString nativeComp (let
-      libPath = lib.concatStringsSep ":" [
-        "${lib.getLib libgccjit}/lib/gcc/${targetPlatform.config}/${libgccjit.version}"
-        "${lib.getLib stdenv.cc.cc}/lib"
-        "${lib.getLib stdenv.glibc}/lib"
-      ];
+      libPath = (lib.concatStringsSep " "
+        (builtins.map (x: ''\"-L${x}\"'') [
+          "${lib.getLib libgccjit}/lib"
+          "${lib.getLib libgccjit}/lib/gcc/${targetPlatform.config}/${libgccjit.version}"
+          "${lib.getLib stdenv.cc.cc}/lib"
```

Comment:
I'm not sure `"${lib.getLib stdenv.cc.cc}/lib"` needs to be included. I believe adisbladis copied these based on discussion in the emacs-overlay repo from a comment I left, and I at the time didn't realize the gcc libraries did not need to be in LIBRARY_PATH. I tested without, and it seems to be fine.